x/pricefeed/simulation/operations/msg.go

```diff
@@ -98,6 +83,51 @@ func SimulateMsgUpdatePrices(keeper keeper.Keeper) simulation.Operation {
 	}
 }
 
+func getStartPrice(marketID string) (startPrice sdk.Dec) {
+	switch marketID {
+	case "btc:usd":
+		return sdk.MustNewDecFromStr("7000")
+	case "bnb:usd":
+		return sdk.MustNewDecFromStr("15")
+	case "xrp:usd":
+		return sdk.MustNewDecFromStr("0.25")
+	}
+	return sdk.MustNewDecFromStr("100")
+}
+
+func getIncrement(marketID string) (increment sdk.Dec) {
+	startPrice := getStartPrice(marketID)
+	divisor := sdk.MustNewDecFromStr("20")
```

Comment:
Increasing `divisor` decreases volatility. E.g. setting to `100` instead of `20` will result in less volatility. Nothing to update just noting as an FYI